### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by touilleMan**

### CODE REVIEW

File: libparsec/crates/types/src/local_device_file.rs [hunk 1]
```diff
@@ -153,15 +153,14 @@ impl DeviceFileType {
     }
 }
 
-#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize)]
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
```
Comment (FirelightFlagboy) [inline libparsec/crates/types/src/local_device_file.rs:156]: What is the reason for the `Serialize` to be remove, we no longer want `AvailableDevice` to be serializable ?

Comment (touilleMan) [reply libparsec/crates/types/src/local_device_file.rs:156]: `AvailableDevice` is never serialized (it is only provided to the GUI to list the devices detected on disk)

Comment (FirelightFlagboy) [reply libparsec/crates/types/src/local_device_file.rs:156]: thanks

**Review by FirelightFlagboy — APPROVED** [review]

**Comment by touilleMan** [comment]
included in #4942

**Pull request closed without merging**